api/src/main/java/org/openmrs/User.java

```diff
@@ -148,7 +148,7 @@ public boolean hasRole(String r, boolean ignoreSuperUser) {
 		Set<Role> tmproles = getAllRoles();
 		
 		if (log.isDebugEnabled()) {
-			log.debug("User #" + userId + " has roles: " + tmproles);
+			log.debug("User # {} has roles {}", userId, tmproles);
```

Comment:
please remove whitespace after the `#` and you are missing a `:` after roles to keep the message identical